core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -95,6 +98,39 @@ public SetupWizard() {
 
     private static final Logger LOGGER = Logger.getLogger(SetupWizard.class.getName());
 
+    private static final String ADMIN_INITIAL_API_TOKEN_PROPERTY_NAME = SetupWizard.class.getName() + ".adminInitialApiToken";
+
+    /**
+     * This property determines the behavior during the SetupWizard install phase concerning the API Token creation 
+     * for the initial admin account.
+     * The behavior depends on the provided value:
+     * - true
+     *      A token is generated using random value at startup and the information is put 
+     *      in the file "$JENKINS_HOME/secrets/initialAdminApiToken".
+     * - [2-char hash version][32-hex-char of secret], where the hash version is currently only 11. 
+     *      E.g. 110123456789abcdef0123456789abcdef.
+     *      A fixed API Token will be created for the user with that plain value as the token.
+     *      It is strongly recommended to use it to generate a new one (random) and then revoke it.
+     *      See {@link ApiTokenProperty#generateNewToken(String)} and {@link ApiTokenProperty#revokeAllTokensExceptOne(String)}
+     *      for scripting methods or using the web API calls: 
+     *      /user/[user-login]/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken and 
+     *      /user/[user-login]/descriptorByName/jenkins.security.ApiTokenProperty/revokeAllExcept 
+     * - @[file-location] where the file contains plain text value of the token, all stuff explained above is applicable
+     *      The application will not delete the file after read, so the script is responsible to clean up the stuff
+     *
+     * When the API Token is generated using this system property, it's strongly recommended that you are revoking it
+     * during your installation script using the other ways at your disposal so that you have a fresh token
+     * with less traces for your script.
+     *
+     * If you do not provide any value to that system property, the default admin account will not have an API Token,
+     * it's the default behavior.
```

Comment:
As it is, the text contains sentence fragments. It needs a joining mechanism, or it's just as well to remove that last part.
```suggestion
     * If you do not provide any value to that system property, the default admin account will not have an API Token.
```